test/tasks/lcov_test.exs

```diff
@@ -107,6 +109,21 @@ defmodule LcovEx.Tasks.LcovTest do
       assert File.read!("example_umbrella_project/cover/lcov.info") ==
                umbrella_output() <> umbrella_output_2()
     end
+
+    test "mix lcov on umbrella app without the dependency" do
+      refute File.read!("example_umbrella_project/apps/example_project_2/mix.exs") =~ "lcov"
+
+      assert {output, 0} =
+               System.cmd("mix", ["lcov", "apps/example_project_2"],
+                 cd: "example_umbrella_project"
+               )
+
+      assert output =~ "Generating lcov file..."
```

Comment:
Question: should this also refute that the top-level cover file is generated?

Reply:
```suggestion
      assert output =~ "Generating lcov file..."
      refute output =~ "Coverage file created at cover/lcov.info"
```